documentation/modules/auxiliary/gather/zookeeper_info_disclosure.md

```diff
@@ -0,0 +1,108 @@
+## Vulnerable Application
+
+### Description
+
+This module targets Apache ZooKeeper service instances to extract information about the system environment, and service statistics.
+
+### Verification Steps
+
+
+## Scenarios
+
+```
+msf5 > use auxiliary/gather/zookeeper_info_disclosure
+msf5 auxiliary(gather/zookeeper_info_disclosure) > set rhosts 1.3.3.7
+msf5 auxiliary(gather/zookeeper_info_disclosure) > show options
+
+       Name: Apache ZooKeeper Information Disclosure
+     Module: auxiliary/gather/zookeeper_info_disclosure
+    License: Metasploit Framework License (BSD)
+       Rank: Normal
+  Disclosed: 2020-10-14
+
+Provided by:
+  Karn Ganeshen <KarnGaneshen@gmail.com>
+
+Check supported:
+  No
+
+Basic options:
+  Name     Current Setting  Required  Description
+  ----     ---------------  --------  -----------
+  RHOSTS   1.3.3.7          yes       The target host(s), range CIDR identifier, or hosts file with syntax 'file:<path>'
+  RPORT    2181             yes       The target port (TCP)
+  THREADS  1                yes       The number of concurrent threads (max one per host)
+  TIMEOUT  30               yes       Timeout for the probe
+
+Description:
+  Apache Zookeeper server service runs on TCP 2181 and by default, it 
+  is accessible without any authentication. This module targets Apache 
+  ZooKeeper service instances to extract information about the system 
+  environment, and service statistics.
+
+References:
+  https://zookeeper.apache.org/doc/current/zookeeperAdmin.html
+
+
+msf5 auxiliary(gather/zookeeper_info_disclosure) > run
+
+[*] 1.3.3.7:2181       - Dumping environment info...
+[+] 1.3.3.7:2181       - Environment:
+zookeeper.version=3.4.9-1757313, built on 08/23/2016 06:50 GMT
+host.name=localhost.localdomain
```

Comment:
hostname can, os name can be saved to `report_host`
example: https://github.com/rapid7/metasploit-framework/blob/master/lib/msf/core/auxiliary/f5.rb#L133

Reply:
Done. Added reporting for hostname, & os details, both getting recorded in hosts and services respectively.

```
msf5 auxiliary(gather/zookeeper_info_disclosure) > hosts 

Hosts
=====

address        mac  name                      os_name  os_flavor  os_sp  purpose  info  comments
-------        ---  ----                      -------  ---------  -----  -------  ----  --------
1.3.3.7       localhost  Linux                      device         Linux amd64 3.10.0-693.11.6.el7.x86_64

msf5 auxiliary(gather/zookeeper_info_disclosure) > services 
Services
========

host           port  proto  name       state  info
----           ----  -----  ----       -----  ----
1.3.3.7  2181  tcp    zookeeper  open   Apache Zookeeper: 3.4.13-2d71af4dbe22557fda74f9a9b4309b15a7487f03


```

Point to note, not all deployments show os details.

Reply:
Done. Added reporting for hostname, & os details, both getting recorded in hosts and services respectively.

```
msf5 auxiliary(gather/zookeeper_info_disclosure) > hosts 

Hosts
=====

address        mac  name                      os_name  os_flavor  os_sp  purpose  info  comments
-------        ---  ----                      -------  ---------  -----  -------  ----  --------
1.3.3.7       localhost  Linux                      device         Linux amd64 3.10.0-693.11.6.el7.x86_64

msf5 auxiliary(gather/zookeeper_info_disclosure) > services 
Services
========

host           port  proto  name       state  info
----           ----  -----  ----       -----  ----
1.3.3.7  2181  tcp    zookeeper  open   Apache Zookeeper: 3.4.13

```
Point to note, not all deployments show os details.